src/lib/mongoBackend/MongoCommonUpdate.cpp

```diff
@@ -2375,7 +2375,8 @@ static void updateAttrInNotifyCer
 static void deleteAttrInNotifyCer
 (
   ContextElementResponse* notifyCerP,
-  ContextAttribute*       targetAttr
+  ContextAttribute*       targetAttr,
+  const std::string&      actionType
```

Comment:
This parameter is not used. Should be removed.

Reply:
Fixed in https://github.com/telefonicaid/fiware-orion/pull/4186/commits/8d8174d4a1d7a62451f3d480c9032b9f21569831